orders/views.py

```diff
@@ -1,11 +1,44 @@
-from django.views    import View
-from django.http     import JsonResponse
+import json, uuid
 
+from django.http      import JsonResponse
+from django.views     import View
+
+from orders.models   import OrderStatus, Order, OrderItem
 from products.models import Product, ProductImage
-from .models         import OrderItem, OrderStatus
 from users.utils     import user_decorator
 
 class CartView(View):
+    @user_decorator
+    def post(self, request):
+        try:
+            data           = json.loads(request.body)
+            user           = request.user
+            product        = data['product_id']
+            current_amount = 0
+
+            if not Product.objects.filter(id=product).exists():
+                return JsonResponse({'message':'PRODUCT_DOES_NOT_EXIST'}, status=400)
+
+            order, is_created = Order.objects.get_or_create(
+                        user            = user,
+                        order_status_id = OrderStatus.PENDING,
+                        defaults        = {'order_number':uuid.uuid4()}
+                    )
+
+            if OrderItem.objects.filter(order=order, product_id=product).exists():
+                current_amount = OrderItem.objects.get(order=order, product_id=product).amount
+
+            orderitem, is_created = OrderItem.objects.update_or_create(
+                        order      = order,
+                        product_id = product,
+                        defaults   = {'amount':data['amount'] + current_amount},
+                        )
+
+            return JsonResponse({'message':"SUCCESS", "id":orderitem.id},status=200)
```

Comment:
```suggestion
            return JsonResponse({'message':"SUCCESS", "order_item_id":orderitem.id},status=200)
```
어떤것의 id인지 명확히 해주는 것이 좋습니다.

Reply:
넵 수정했습니다!